bindings/java/org/collectd/java/GenericJMXConfConnection.java

```diff
@@ -26,19 +26,20 @@
 
 package org.collectd.java;
 
+import static javax.management.remote.rmi.RMIConnectorServer.RMI_CLIENT_SOCKET_FACTORY_ATTRIBUTE;
```

Comment:
could you please keep the imports in alphabetical order?

Reply:
Done and rebased!